components/monitoring/grafana/base/kustomization.yaml

```diff
@@ -15,6 +15,7 @@ resources:
 - release/
 - performance/
 - pipeline-service/
+- dashboards/
```

Comment:
Can we rename this directory? `dashboards` is too generic IMO. All other directories referenced here contain dashboards.

Reply:
I think that `generic-dashboards` would be a good name for the directory